src/test/resources/de/bsi/secvisogram/csaf_cms_backend/integration/fullworkflow.feature

```diff
@@ -0,0 +1,128 @@
+	* def advisoryId = response.id
+	* def revision = response.revision
+
+	########	
+	#Change workflow state to Review
+	#
+	# Draft, Review, Approved, RfPublication, Published
+	* def workflowStatus = 'Review'
+	Given path apiBase + '/' + advisoryId + '/workflowstate/' + workflowStatus
+	* param revision = revision
+	* header Authorization = 'Bearer ' + accessToken
+	* header Content-Type = 'application/json'
+	When method patch
+	Then status 200
+	
+	# Get new revision after workflow state change
+	Given path apiBase
+	* header Authorization = 'Bearer ' + accessToken
+	* header Content-Type = 'application/json'
+	When method get
+	Then status 200
+	* def filt = function(x){ return x.advisoryId == advisoryId }
+    * def items = get response[*]
+    * def revision = karate.filter(items, filt)[0].revision    
+
+	########	
+	#Change workflow state to Approved
+	#
+	* def workflowStatus = 'Approved'
+	Given path apiBase + '/' + advisoryId + '/workflowstate/' + workflowStatus
+	* param revision = revision
+	* header Authorization = 'Bearer ' + accessToken
+	* header Content-Type = 'application/json'
+	When method patch
+	Then status 200
+	# Get new revision after workflow state change
+	Given path apiBase
+	* header Authorization = 'Bearer ' + accessToken
+	* header Content-Type = 'application/json'
+	When method get
+	Then status 200
+	* def filt = function(x){ return x.advisoryId == advisoryId }
+    * def items = get response[*]
+    * def revision = karate.filter(items, filt)[0].revision    
+
+	########	
+	#Change workflow state to RfPublication
+	#
+	* def workflowStatus = 'RfPublication'
+	Given path apiBase + '/' + advisoryId + '/workflowstate/' + workflowStatus
+	* param revision = revision
+	* header Authorization = 'Bearer ' + accessToken
+	* header Content-Type = 'application/json'
+	When method patch
+	Then status 200
+	# Get new revision after workflow state change
+	Given path apiBase
+	* header Authorization = 'Bearer ' + accessToken
+	* header Content-Type = 'application/json'
+	When method get
+	Then status 200
+	* def filt = function(x){ return x.advisoryId == advisoryId }
+    * def items = get response[*]
+    * def revision = karate.filter(items, filt)[0].revision 
+
+	########	
+	#Change workflow state to Published
+	#
+	* def workflowStatus = 'Published'
+	Given path apiBase + '/' + advisoryId + '/workflowstate/' + workflowStatus
+	* param revision = revision
+	# final, interim, draft
+	* param documentTrackingStatus = 'Final'
+	* header Authorization = 'Bearer ' + accessToken
+	* header Content-Type = 'application/json'
+	When method patch
+	Then status 200
+		
+	# Get new revision after workflow state change
+	Given path apiBase
+	* header Authorization = 'Bearer ' + accessToken
+	* header Content-Type = 'application/json'
+	When method get
+	Then status 200
+	* def filt = function(x){ return x.advisoryId == advisoryId }
+    * def items = get response[*]
+    * def revision = karate.filter(items, filt)[0].revision 
```

Comment:
Same here

Reply:
Fixed in https://github.com/secvisogram/csaf-cms-backend/pull/159/commits/2f18434ca7af4783e402c0b5ff70be17bd4e9808